Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/view/TimePickerDialog.java

```diff
@@ -0,0 +1,84 @@
+package rdatu.android.cyscorpions.com.projectplanner.view;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+import android.widget.TimePicker;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Created by rayeldatu on 7/28/15.
+ */
+public class TimePickerDialog extends DialogFragment implements android.app.TimePickerDialog.OnTimeSetListener {
+
+    public static final String EXTRA_TIME = "time";
+    private String mStringTime;
+    private Callbacks mCallbacks;
+
+
+    public TimePickerDialog() {
```

Comment:
For clean code, we can remove this constructor since it does not do any special operations. By default all objects has a ```Object()``` default constructor.